Contributions/CheckList_for_Safety_Claims_on_a_Generic_Linux_System.md

```diff
@@ -0,0 +1,269 @@
+
+The risk is that one might overlook aspects which are paramount to safety, but typically deemed irrelevant for linux users and even to kernel developers they might appear as very niche.
+The list here provided (as-is) does not claim to be sufficient, nor correct (Linux evolves rapidly), and it is strongly advised to confirm that the statements made are still correct and relevant.
+
+However, it is an attempt to bring the attention to aspects that might not be immediately obvious.
+One immediate example of such an aspect is the fact that it is practically impossible to talk about safety without defining the reference hardware platform.
+
+The entire document will mostly avoid entering in the details of both hardware implementation and Linux internals, beyond what is strictly required to justify each individual item of the checklist.
+Similarly, it won't provide extensive justification of why something might be relevant, from a safety perspective.
+
+All of the background justification, related to hardware (ARM64), Linux kernel architecture, and allocation of safety requirements, they can be found in the documents mentioned in the [References](#References) section.
+
+Last but not least, this document is not meant to provide solutions to the problems listed: solutions are often heavily contextualised, and must be designed to suit specific needs, in terms of use cases, safety requirements, system design.
+
+The evaluation of the very same mitigation strategy can range from overkill to insufficient, depending on where/how it is applied.
+
+
+## **Structure of the document**
+- The next two sections are a brief - but necessary - premise about hardware and toolchains: Linux is a code-base, but what gets deployed is a combination of hardware and binary code. This aspect cannot be overlooked.
+- In the following section, the actual list unfolds.
+
+
+
+## **Hardware Platform**
+
+When a system is expected to meet certain safety requirements, these requirements are typically allocated to some high-level functionality, which is what constitutes the actual focus of interest in a safety context.
+
+The system is then iteratively broken down into components, and safety requirements are percolated accordingly. At each iteration, the components having to support safety requirements may be further broken down, until the complexity of the components identified is deemed to be treatable.
+
+At the very bottom of the stack of components lies the hardware. Hardware components may be qualified to a given safety level. Or not.
+
+The level of safety qualification (or lack of) presented by the relevant hardware components can affect significantly the overall strategy used for the qualification of the entire system.
+
+For example, by using chips that have been both designed and produced following qualified processes, implemented with qualified tools, it is possible to make assumptions about hardware safety.
```

Comment:
```suggestion
For example, by using chips that have been both designed and produced following qualified processes, and implemented with qualified tools, it is reasonable to make assumptions about hardware safety.
```

Reply:
In this case I meant "possible", as in "legitimate". As objectie enabler. I feel that "reasonable" waters down the strenght I inentionaly wanted to use. Assuming that I'm attributing the correct meaning to theose terms.